Z/z3/build_tarballs.jl

```diff
@@ -36,9 +35,12 @@ fi
 """
 
 # Bash recipe for building across all platforms
+# Patches Z3 to work around https://github.com/Z3Prover/z3/issues/6890 (closed but not fixed!)
```

Comment:
This is better placed right above the line doing the patching, and also a description of what the patch does - e.g. why is that sed command patching it? The linked issue doesn't mention this as a fix at all.

Reply:
I linked to a new issue that documents more details about the problem, and moved the comment down.